test/issuance.test.js

```diff
@@ -84,6 +84,13 @@ describe("commitments parsing and caching", () => {
         expect(v11H === commitments.H).toBeTruthy();
     });
 
+    test("parse correctly (v1.01)", () => {
```

Comment:
It would be worth adding some tests that should fail, such as: 
- if the signature is incorrect and fails to verify
- if the verification key is invalid so that it fails to parse

Reply:
done